### ISSUE TITLE
Manually move user focus on 2nd article view.

### ISSUE THREAD

**Issue by dr3** [bbc/simorgh#1303]
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/issues/1302 the loading state is delayed to reduce visual bounce. In doing this, there is a chance the loading screen is never seen. This causes issues when transferring from 1 article to another seamlessly, as react will try to convert the article, not reseting the user focus.

**Describe the solution you'd like**
Manually move the user focus when transitioning to a loading screen or new article.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by jamesbhobbs** [comment]
Blocked by/subsequent to https://github.com/bbc/simorgh/issues/1302

**Comment by dr3** [comment]
Closing in favour of https://github.com/bbc/simorgh/issues/1302 which contains both delayed loading and moving focus, as they are tied togeather
